services/api-server/src/simcore_service_api_server/modules/webserver.py

```diff
@@ -164,16 +121,20 @@ async def get_project(self, project_id: UUID) -> Project:
         return Project.parse_obj(data)
 
     async def list_projects(self, solver_name: str) -> List[Project]:
+        # TODO: pagination?
```

Comment:
Just thinking. Is it not better to add a parameter for calls from backend to disable pagination? By that I mean to respond with all the items from the backend side.

Reply:
Yes. We do not really need pagination here. It was just a fleeting thought. What we DO need (and is part of #2344 ) is adding generic filter functionality during list.